content/department-processes/team-onboarding/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Onboarding - What the team needs to know and access"
+ready: true
+---
+
+This is a guide to the Tech Department and how it works. It contains contact details, important links to documents we use and software that are useful.
+
+### General
+
+#### What recruits need to know
+Firstly, read through {{% contentlink "/department-processes/what-recruits-need-to-know/_index.md" %}}
+
+#### Syllabus
+Find the tech repo with all (or at least most) curriculum documents, assignments, processes and deadlines here: https://umuzi-org.github.io/tech-department/
+
+#### Slack 
+Please check your slack messages regularly throughout the day (at least 3 times per day).
```

Comment:
Please also check your slack messages first thing in the morning before you do your first task for the day.